src/connector/src/state.rs

```diff
@@ -19,6 +19,10 @@ use bytes::Bytes;
 use log::error;
 use risingwave_storage::storage_value::StorageValue;
 use risingwave_storage::{Keyspace, StateStore};
+#[allow(unused_imports)]
+use serde::{Deserialize, Serialize};
+
+use crate::SourceSplit;
```

Comment:
It should really be called SplitMetadata or SplitState. The current is not very expressive.

Reply:
fixed